### DISCUSSION TITLE
Why did you choose to implement your tools only in cpp?

### DISCUSSION THREAD

**Discussion by giladmaya** [speedb-io/speedb#302] [Q&A]
Some of the tools included in speedb include a lot of code dedicated to input parsing. Why did you implement them with CPP instead of choosing a scripting language like python?

**Comment by mrambacher** [comment]
@giladmaya Can you give an example of the tools you are referring to?  More likely than not, they were developed in a language natural to the developer rather than the "best" language for the tool (I for one am lousy at Python).  Another reason might be to reduce external dependencies (not requiring python on the target machine).

Care to take a crack at porting one to another language?  PRs are always welcome!
